src/main/java/com/hillel/items_exchange/dto/AdvertisementTitleDto.java

```diff
@@ -0,0 +1,19 @@
+package com.hillel.items_exchange.dto;
+
+import com.hillel.items_exchange.model.Location;
+import lombok.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@EqualsAndHashCode
```

Comment:
You should exclude **advertisementId** from equals and hashcode

Reply:
Yeah. You are right.